ui/shared/SeparatorWithIcon.qml

```diff
@@ -0,0 +1,48 @@
+import QtQuick 2.13
+import QtGraphicalEffects 1.13
+import "../imports"
+
+Item {
+    property int iconMargin: Style.current.padding
+    property alias icon: icon
+    readonly property int separatorWidth: (parent.width / 2) - (icon.height / 2) - iconMargin
+    width: parent.width
+    height: icon.height
+    
+
+    Rectangle {
+        id: separatorLeft
```

Comment:
We have a Separator component that you can use. It already has the right border, height, etc

Reply:
Is it `/shared/separator.qml`? I did try to use this, but It didn't have an icon and adding one proved a bit challenging.

Reply:
Nice, updated.